.travis.yml

```diff
@@ -23,6 +23,11 @@ install:
       cd protoc
       wget https://github.com/google/protobuf/releases/download/v3.2.0/protoc-3.2.0-${TRAVIS_OS_NAME}-x86_64.zip
       unzip protoc-3.2.0-${TRAVIS_OS_NAME}-x86_64.zip
+      cd include/google
```

Comment:
Pulling everything in generates a whole bunch of errors from the other checks (`misspell` in particular), so it seemed easier to stick to the 2 files we need and only have two exceptions (`cancelled`) for the lint checks.